pom.xml

```diff
@@ -336,6 +350,7 @@
 					<links>
 						<link>https://javaee.github.io/javaee-spec/javadocs/</link>
 					</links>
+                                        <doclint>none</doclint>
```

Comment:
Why this change?

Reply:
I changed it just now to do all the checks but `-reference` category, because it can't find Lombok-generated builders otherwise and errors out